src/platform/forms/tests/save-in-progress/actions.unit.spec.js

```diff
@@ -320,6 +335,27 @@ describe('Schemaform save / load actions:', () => {
         );
       });
     });
+    it('dispatches a success if the form is loaded', () => {
```

Comment:
Like above, this is specifically testing that the _override_ works, right? Can we make the test name reflect this?